Projects/vbr_core_examples/CB_019_backstress_maxwell.m

```diff
@@ -0,0 +1,42 @@
+function VBR = CB_019_backstress_maxwell
+        %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+        % CB_019_backstress_maxwell
+        %
+        % Demonstration of a Maxwell lmodel with steady-state viscosity
+        % of the backstress model using the analytical_andrade model
+        %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+        
+        VBR.in.anelastic.methods_list = {'andrade_analytical'};
+        VBR.in.anelastic.andrade_analytical.Beta = 0; % turn off anelastic part of Andrade model and turns it into a Maxwell model
```

Comment:
I **just** added a plain maxwell model if you want to use it. You'd have to merge this branch with the `main` upstream branch and then you should be able to delete the `Beta` line and find/replace `andrade_analytical` with `maxwell_analytical` in this file.

Here's the docs for the new function: https://vbr-calc.github.io/vbr/vbrmethods/anel/maxwellanalytical/

I haven't tested it too much, so if you do this and see issues, definitely let me know!

Reply:
(you can ignore this comment and leave the code as-is too)

Reply:
i just merged this branch with the main branch, you should be able to use maxwell here now.